functions_task_is_prime_function.py

```diff
@@ -0,0 +1,15 @@
+# Write a function is_prime that takes 1 argument - a number from 2 to 1000,
+# and returns True if it is a prime number, and False otherwise.
+
+def is_prime(number_checker: int) -> int:
+    if 2 <= number_checker <= 1000:
+        return True
+    return False
```

Comment:
please read about prime numbers. It is not a correct solution
https://en.wikipedia.org/wiki/Prime_number